### PR TITLE
Cleanup network page

### LINKED ISSUE #43: Clean up layout of Network page
* Remove "IP Addresses(s)" title - just list the addresses
* Simplify layout so each interface section looks something like this:
image
* When the test button is pushed, pop up a Bootstrap modal and show the test results in there
* Use a animated spinner gif to indicate a test is in progress

### PR DESCRIPTION
**Pull request by Davo36**
Closes #43

### CODE REVIEW

**Comment by mjs** [comment]
This looks and works so much better. Thanks.

Can you make it so the box for each interface takes up the full page width (up to some limit, say 600px, so it doesn't go crazy wide on PC monitors)? That would make it look much better on mobile. Right now it looks like this on my phone:

image

Everything would fit better if the boxes filled the page.

**Comment by mjs** [comment]
A little more padding around the ping output would be nice too. It currently looks like this on my phone:

image

File: html/network.html [hunk 1]
```diff
@@ -72,6 +103,71 @@ <h5 class="card-title"><strong>Output</strong></h5>
...
+            // Disable test button while waiting for response
+            var button = document.getElementById("interface-" + interfaceName + "-test");
+            button.disabled = true;
+            // Also make it so the modal can't be closed while the test is running.
+            modal.find('button#closeButton').prop("disabled", true)
+            modal.find('button#crossCloseButton').prop("disabled", true)
+
+            // Show spinner
+            modal.find('div#testingSpinner').prop("hidden", false)
+            // document.getElementById("testingSpinner").hidden = false
```
Comment (mjs) [inline html/network.html]: delete

File: html/network.html [hunk 2]
```diff
@@ -72,6 +103,71 @@ <h5 class="card-title"><strong>Output</strong></h5>
...
+
+            fetch(
+              "/interface-status/" + interfaceName,
+              {method: "GET"}
+            ).then(function(response) {
+              return response.json();
+            }).then(function(json) {
+
+              // Hide spinner
+              // document.getElementById("testingSpinner").hidden = true
```
Comment (mjs) [inline html/network.html]: delete

File: html/network.html [hunk 3]
```diff
@@ -72,6 +103,71 @@ <h5 class="card-title"><strong>Output</strong></h5>
...
+                modal.find('h4#statusIndicator').html("<h4>&#x2717;</h4>")
+                // modal.find('.modal-body').html(json["result"] + "<h4 class=\"mt-3\">&#x2717;</h4>")
+              } else if (state === "up") {
+                // Check mark
+                modal.find('h4#statusIndicator').html("<h4>&#x2714;</h4>")
+                // modal.find('.modal-body').html(json["result"] + "<h4 class=\"mt-3\">&#x2714;</h4>")
+              } else {
+                // Question mark
+                modal.find('h4#statusIndicator').html("<h4>&#x003F;</h4>")
+                // modal.find('.modal-body').html(json["result"] + "<h4 class=\"mt-3\">&#x003F;</h4>")
```
Comment (mjs) [inline html/network.html]: delete all these commented out bits

Comment (Davo36) [reply html/network.html]: Ok, doing this now.  Got the cards to be full screen up to 600px.  Working on the output padding...

**Comment by Davo36** [comment]
That was easy enough to do.  You're right, it looks better now.

**Pull request merged**